catalog/USGS/USGS_NLCD_RELEASES_2021_REL_TCC_v2021-4.jsonnet

```diff
@@ -0,0 +1,317 @@
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local id = 'USGS/NLCD_RELEASES/2021_REL/TCC/v2021-4';
+local version = 'v2021-4';
+local subdir = 'USGS';
+local basename = std.strReplace(id, '/', '_');
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+local license = spdx.proprietary;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'USFS Tree Canopy Cover ' + version + ' ' + '(CONUS and OCONUS)',
+  'gee:skip_indexing': true,
+  version: version,
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    This product is part of the Tree Canopy Cover (TCC) data suite. It includes modeled TCC, standard error (SE), and 
+    National Land Cover Database’s (NLCD) TCC data for each year. TCC data produced by the the United States Department 
```

Comment:
please use the standard ASCII apostrophe in "Database's"

Reply:
Fixed the apostrophe